src/platform/EFR32/EFR32Config.cpp

```diff
@@ -533,3 +534,90 @@ void EFR32Config::RepackNvm3Flash(void)
 } // namespace Internal
 } // namespace DeviceLayer
 } // namespace chip
+
+extern "C" {
```

Comment:
not sure if this is the right location to have moved those. but other than this it lgtm